pkg/envtest/server.go

```diff
@@ -46,54 +46,66 @@ It's possible to override some defaults, by setting the following environment va
 
 */
 const (
-	envUseExistingCluster  = "USE_EXISTING_CLUSTER"
-	envKubeAPIServerBin    = "TEST_ASSET_KUBE_APISERVER"
```

Comment:
Did you mean to entirely remove these? That makes this PR a breaking change, in a subtile way. They are also still documented in `pkg/envtest/server.go`

Reply:
they're still used, just lower-down in the code.  These constants are private, so it's not breaking to remove them here -- they're actually just duplicates from `testing/internal` from when we couldn't tweak testing_frameworks directly

Reply:
(FWIW, they're now used natively in the binpath finder, which we were just short-circuiting before because we couldn't make it do what we wanted it to)